envs/quantile.yaml

```diff
@@ -3,17 +3,17 @@ channels:
  - bioconda
  - ebi-gene-expression-group
 dependencies:
- - perl-atlas-modules=0.3.1
+ - perl-atlas-modules >=0.3.1
  - libopenblas=0.3.3
  - r-ff=2.2-14
  - r-bit=1.1-15.2
  - r-bit64=0.9-7.1
- - bioconductor-limma=3.42.0 
- - bioconductor-lumi=2.38.0
- - bioconductor-oligo=1.50.0
- - bioconductor-oligoclasses=1.48.0
- - bioconductor-biobase=2.46.0
- - bioconductor-arrayqualitymetrics=3.42.0
+ - bioconductor-limma
```

Comment:
Any particular reason for removing version.. ? it always picks latest version of not specified and in future if may have conflict